src/CarbonAware.WebApi/test/integrationTests/mocks/WattTimeServerMocks.cs

```diff
@@ -0,0 +1,140 @@
+using System.Net;
+using System.Text.Json;
+using WireMock.Server;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using System.Net.Mime;
+using CarbonAware.Tools.WattTimeClient.Model;
+using CarbonAware.Tools.WattTimeClient.Constants;
+
+namespace CarbonAware.Tools.WattTimeClient
+{
+    /// <summary>
+    /// A utilities static class for Watt Time
+    /// </summary>
+    public static class WattTimeServerMocks
```

Comment:
Can you explain what the goal is with keeping `WattTimeDataSourceMocker` and this class separate?  This class feels like it is the actual Mocker and just needs the `overrideWebAppFactory` added to it.

Reply:
We separated it out due to a separation of labor. I'm of the opinion that the WattTimeServerMocks class is doing the meat of the work, but in terms of showcasing the framework, this might be more instructive.